src/components/ButtonPanel.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import '../ButtonPanel.css';
+
+const ButtonPanel = () => (
+  <div className="calculator-grid">
+    <button type="button">AC</button>
+    <button type="button">+/-</button>
+    <button type="button">%</button>
+    <button type="button">÷</button>
+    <button type="button">7</button>
+    <button type="button">8</button>
+    <button type="button">9</button>
+    <button type="button">X</button>
+    <button type="button">4</button>
+    <button type="button">5</button>
+    <button type="button">6</button>
+    <button type="button">-</button>
+    <button type="button">1</button>
+    <button type="button">2</button>
+    <button type="button">3</button>
+    <button type="button">+</button>
+    <button type="button" className="span-two">0</button>
+    <button type="button">.</button>
+    <button type="button">=</button>
+  </div>
```

Comment:
**[Required]** Buttons component should be grouped as instructed in the requirements.[Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/buttonPanel/buttons.png)

For example, 1  2, 3, and +  should be grouped in a div.